### ISSUE TITLE
New lxml no longer supports Python 3.4

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbem#1807]
The Travis runs fail on Python 3.4 with latest package levels, e.g. from https://travis-ci.org/pywbem/pywbem/jobs/573094054:
```
Collecting lxml>=4.2.4 (from -r dev-requirements.txt (line 16))
  Downloading https://files.pythonhosted.org/packages/e1/f5/5eb3b491958dcfdcfa5daae3c655ab59276bc216ca015e44743c9c220e9e/lxml-4.4.0.tar.gz (4.5MB)
    ERROR: Complete output from command python setup.py egg_info:
    ERROR: This lxml version requires Python 2.7, 3.5 or later.
    ----------------------------------------
ERROR: Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-1_x1_l4_/lxml/
```

**Comment by andy-maier** [comment]
It turns out that lxml 4.4.0 still installed on Python 3.4, whereas lxml 4.4.1 requires at least Python 3.5.
The solution for us is to pin lxml to 4.4.0.

**Comment by andy-maier** [comment]
Addressed in PR #1808 for master.

**Comment by andy-maier** [comment]
Addressed in PR #1809 for pywbem 0.14.5
